.scripts/prune_registry.py

```
@@ -406,10 +414,14 @@ def main():
 
     print()
     if DRY_RUN:
-        print(f"📊 Summary: Would delete {deleted_count} version(s) across {len(packages)} package(s)")
+        print(
+            f"📊 Summary: Would delete {deleted_count} version(s) across {len(packages)} package(s)"
+        )
         print(f"   ℹ️ Run without DRY_RUN=true to actually delete these versions")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)